src/Microsoft.IdentityModel.Tokens/Validators.cs

```diff
@@ -38,6 +38,34 @@ namespace Microsoft.IdentityModel.Tokens
     /// </summary>
     public static class Validators
     {
+        /// <summary>
+        /// Validates if a given algorithm for a <see cref="SecurityKey"/> is acceptable
+        /// </summary>
+        /// <param name="validationParameters"><see cref="TokenValidationParameters"/> required for validation.</param>
+        /// <param name="algorithm">The algorithm being considered if it is valid</param>
+        /// <param name="securityKey">The <see cref="SecurityKey"/> that signed the <see cref="SecurityToken"/>.</param>
+        /// <param name="securityToken">The <see cref="SecurityToken"/> being validated.</param>
+        /// <returns><c>true</c> if the algorithm is acceptable</returns>
+        public static bool ValidateAcceptedAlgorithms(TokenValidationParameters validationParameters, string algorithm, SecurityKey securityKey, SecurityToken securityToken)
```

Comment:
can we have a similar as the other validators?
Parameters of interest first - algorithm, securitykey
SecurityToken, 
TokenValidationParameters last,

Reply:
good idea